### DISCUSSION TITLE
Multiple Choice Filter: el filtro de usuario por job title, se muestran todos los usuarios de cotalker?

### DISCUSSION THREAD

**Discussion by ValeMP** [Cotalker/documentation#121] [2. Questions and Answers] [answered]
Hola!

Estoy usando un survey que tiene un componente de multiple choice que muestra los usuarios haciendo filtro por job Title. Quería saber si debería mostrar a todos los usuarios de cotalker que tienen ese job Title, o solo a los usuarios del canal que tienen ese job title.

gracias!

**Comment by acedward** [comment, marked as answer]
@ValeMP quienes ven dependen de la configuración de la Compañía, pero siempre te muesta siempre todos **tus contactos** en Cotalker.


Ahora, **tus contactos** dependen de la opción global de visiblidad de usuarios la compañía:
1. Ves solo tus contactos directos (a quienes tienes en otros canales) 
2. Visibilidad global y puedes ver a todos.

**Reply by acedward** [reply]
Hemos creado un issue para mejorar la documentación respecto a este punto:
https://github.com/Cotalker/documentation/issues/124

**Reply by ValeMP** [reply]
bacan! sii trate de revisar antes la documentacion pero no lo encontré!

**Reply by acedward** [reply]
No hay más feedback. Voy a cerrar.
